iot/api/README.md

```diff
@@ -0,0 +1,38 @@
+# Cloud IoT Core NodeJS Samples
+This folder contains NodeJS samples that demonstrate an overview of the
+Google Cloud IoT Core platform.
+
+## Quickstart
+
+1. Install the gCloud CLI as described in [the device manager guide](https://cloud-dot-devsite.googleplex.com/iot/docs/device_manager_guide).
```

Comment:
There is not, I also do not have the timeline for the whitelist being removed but it should be shorter than most of our whitelist alphas.